### DISCUSSION TITLE
Online vector maps support?

### DISCUSSION THREAD

**Discussion by Psijic** [mapsforge/mapsforge#1441] [Q&A] [answered]
I see, OSM online raster map is supported here but didn't find any signs of the vector ones while Vtm supports many of them. Also - what are the main differences between Vtm and mapsforge except of OpenGL usage, can they be merged? I see even release date and versions are same.

**Comment by devemux86** [comment, marked as answer]
Mapsforge does not support online vector tiles.
There are some "tile server" examples, see the [applications list](https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Applications.md).

Mapsforge and VTM have different map engines.
VTM uses OpenGL, has extra features and can read Mapsforge maps.
